### PR TITLE
HoTT-UF core library with cubical machinery hidden

### PR DESCRIPTION
**Pull request by martinescardo**
An interface file for users who wish to compute with funext, propositional truncation and univalence in cubical Agda with the cubical machinery hidden.

### CODE REVIEW

File: Cubical/HoTT-UF.agda [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+notation.
+
+The point is that function extensionality, propositional truncation
+and univalence compute (an example is given below).
+
+For the moment, this requires the development version of Agda.
+
+-}
+
+{-# OPTIONS --cubical --exact-split --safe #-}
```
Comment (mortberg) [inline Cubical/HoTT-UF.agda]: What is `--exact-split` and `--safe`? Should these be added to the other files?

File: Cubical/HoTT-UF.agda [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  c : pr₁ b ≡ five
+  c = refl
+
+{-
+
+If we had funExt as a postulate, then the definition of c would not
+type check. Moreover, the term pr₁ b would not evaluate to five, as it
+does with the cubical type theory implementation of funext.
+
+TODO. A similar computational example with univalence.
```
Comment (mortberg) [inline Cubical/HoTT-UF.agda]: Do you have any idea for this?

**Comment by mortberg** [comment]
I'm merging this now. I guess we can still discuss my two questions afterwards. Thanks a lot for contributing!

File: Cubical/HoTT-UF.agda (hunk 1 above)
Comment (martinescardo) [reply Cubical/HoTT-UF.agda]: * [safe](https://agda.readthedocs.io/en/v2.5.4.2/language/safe-agda.html): "By using the command-line option --safe, a user can specify that Agda should ensure that features leading to possible inconsistencies should be disabled."
* [exact split](https://agda.readthedocs.io/en/v2.5.4/tools/command-line-options.html): "Require all clauses in a definition to hold as definitional equalities ". See [case trees](https://agda.readthedocs.io/en/v2.5.4/language/function-definitions.html#case-trees). In practice you get this if you construct your terms using case split in the emacs mode.

Comment (mortberg) [reply Cubical/HoTT-UF.agda]: Thanks! I'll add `--safe` to all of the files.

**Pull request merged**